astropy/time/core.py

```diff
@@ -408,6 +409,40 @@ def now(cls):
 
     info = TimeInfo()
 
+    @classmethod
+    def strptime(cls, time_string, format_string):
+        """
+        Parse a string to a Time according to a format specification.
+        See time.strptime documentation for format specification.
+
+        >>> Time.strptime('2012-06-30T23:59:60', '%Y-%m-%dT%H:%M:%S')
```

Comment:
Use an example date format that is not already supported by `Time`, e.g. `'%Y-%b-%d %H:%M:%S'`.